### PR TITLE
Upgrade gradle to 4.1

### PR DESCRIPTION
**Pull request by mateuszmarek**

### CODE REVIEW

**Comment by ThePnx** [comment]
I would like to take this opportunity to kindly remind you about #49

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Sun Aug 06 14:43:40 CEST 2017
+#Mon Aug 07 22:43:11 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
```
Comment (ThePnx) [inline gradle/wrapper/gradle-wrapper.properties]: What about `distributionUrl` ?

**Review by ThePnx — CHANGES_REQUESTED** [review]

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (mateuszmarek) [reply gradle/wrapper/gradle-wrapper.properties]: good catch!

Comment (mateuszmarek) [reply gradle/wrapper/gradle-wrapper.properties]: fixed

**Review by ThePnx — APPROVED** [review]

**Pull request merged**